readme.md

```diff
@@ -106,18 +106,25 @@ When merging into a newer MPS version, follow these steps.
 The example assumes we merge `mps2021.1` into `mps2021.2`.
 
 1. Open the _target_ MPS version (2021.2) on the _target_ branch (`mps2021.2`).
-2. Merge the _source_ branch (`mps2021.1`) into your _current_ branch (`mps2021.2`).
-3. Double-check `gradle.properties` still contains the proper entries for:
+2. Checkout a new branch off of the _target_ branch (`mps2021.2`), named `mps2021.2-migration` (or a name that's as least as good).
+3. Merge the _source_ branch (`mps2021.1`) into your _current_ branch (`mps2021.2-migration`).
+4. Double-check `gradle.properties` still contains the proper entries for:
    * `mpsVersionSuffix` should be the _target_ MPS version (`2021.2`)
    * `mpsVersion` full _target_ MPS version (`2021.2.6`)
    * `mpsExtensionsVersion` latest version of [MPS-extensions](https://jetbrains.github.io/MPS-extensions/) for the _target_ MPS version (`2021.2.2631.1360a64`)
-4. Run the Migration Assistant.
-5. Run all tests.
-6. Update the build model.
-7. Repeat steps 3-6 for the test projects, residing in `test-project/` and `test-project-externalLib/`.
-  **Note** Before opening these projects in MPS:
+5. Run the Migration Assistant.
+6. Run all tests. (*TODO* Some elucidation on how to do this precisely, and what tests should absolutely not fail, and what can be safely ignored.)
+7. Update the build model. (*TODO* Some elucidation might be required here as well.)
```

Comment:
I'm not sure we need to explain this in further detail: Everybody who uses MPS should know the drill.